ETL.md

```diff
@@ -0,0 +1,68 @@
+# Green Commons ETL
+
+## Introduction
+
+These are the Extract-Transform-Load (ETL)
+scripts for importing books and other resources
+into the Green Commons database.
+
+We are using the [`kiba`] gem
+to set up ETL pipelines.
+
+[`kiba`]: https://github.com/thbar/kiba
+
+The basic format of one kiba ETL script is as follows:
+
+```ruby
+# lib/etl/example_etl_script.etl
+#
+# Common files required by all processes.
+require_relative 'common'
+
+# Define a source of data to import. In this case, a directory of files.
+# a source file needs to only have a constructor and `.each` method.
+source(ProcessAllFilesInLocalFolder, 'lib/etl/*.epub')
+
+# How to transform one of the above source items. Expected output is a hash.
+transform(TransformEpub)
+
+# What to do with the hash resulting from the above transformation.
+# Here, we are using the hash attributes to create a new database record.
+destination(CreateNewResourceRecord)
+```
+
+For more information, see the [`kiba`] `README.md` file.
+
+## Usage
+
+Then run the rake task to import the records,
+transform them,
+and load them into the database:
+
+```
+bundle exec rake etl:import_local_epub
```

Comment:
Should this perhaps be a more generic rake task? I don't know if this needs to be addressed in this PR but it's specific to an epub, so this command might turn into multiple commands with the addition of more resource types.

Alternatively, should we just add a note requesting developers create a separate rake task for any new resource type they add to the system?